build/travis-deploy.sh

```diff
@@ -37,10 +37,21 @@ if [ -n "$TRAVIS_TAG" ]; then
     make push-release RELEASE_TAG="ppc64le-$TRAVIS_TAG" GOARCH=ppc64le
     make clean RELEASE_TAG="ppc64le-$TRAVIS_TAG" GOARCH=ppc64le
     echo "Pushing manifest on Travis"
-    make push-manifest RELEASE_TAG="$TRAVIS_TAG"
+    make push-manifest-release RELEASE_TAG="$TRAVIS_TAG"
     exit 0
 fi
 
 # Push image tag format: COMMIT
 echo "Running push build on Travis"
-make push
+make push GOARCH=amd64
```

Comment:
It might make sense to define a GOARCHES variable and loop over that one in this script, so adding a new arch is a single line change.

Reply:
Ah yea that'll make more sense. I also found a few bugs with the `push-release` where it only pushes amd64 and not the other arches.

Reply:
Ah yea that makes sense.